test_wazuh/test_fim/test_basic_usage/test_basic_usage_new_dirs.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (C) 2015-2019, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import shutil
+
+import pytest
+
+from wazuh_testing.tools import PREFIX
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.fim import DEFAULT_TIMEOUT, LOG_FILE_PATH, generate_params, \
+    regular_file_cud, check_time_travel
+
+
+# Marks
+
+pytestmark = pytest.mark.tier(level=0)
```

Comment:
Test should only run in linux and windows since there is no realtime or whodata in other OS